libs/ledger-live-common/src/hw/index.ts

```diff
@@ -17,30 +21,47 @@ export type Discovery = Observable<DeviceEvent>;
 export type TransportModule = {
   // unique transport name that identify the transport module
   id: string;
-  // open a device by an id, this id must be unique across all modules
-  // you can typically prefix it with `something|` that identify it globally
-  // returns falsy if the transport module can't handle this id
-  // here, open means we want to START doing something with the transport
-  open: (id: string) => Promise<Transport> | null | undefined;
+  /*
+   * Opens a device by an id, this id must be unique across all modules
+   * you can typically prefix it with `something|` that identify it globally
+   * returns falsy if the transport module can't handle this id
+   * here, open means we want to START doing something with the transport
+   *
+   * @param deviceId
+   * @param timeoutMs Optional timeout that can be used by the Transport implementation when opening a communication channel
+   * @param context Optional context to be used in logs/tracing
+   */
+  open: (
+    deviceId: string,
```

Comment:
we may want to keep calling it "id" because it's not a "deviceId" as in "device id of the transport" but it's the userland that defines whatever the semantic of this "id" is. (LLM/LLD/cli/tools,.. can all decide how they decouple the id and the transport instance)

Reply:
You're right ! I put back `id` ;)